python/ray/tests/test_autoscaler.py

```diff
@@ -320,6 +320,8 @@ def update_nodes(self):
     SMALL_CLUSTER, **{"available_node_types": TYPES_A, "head_node_type": "empty_node"}
 )
 
+DUMMY_IDLE_DURATION_S = 3
```

Comment:
Can you add some comments for `DUMMY_IDLE_DURATION_S`? For example, explain when `DUMMY_IDLE_DURATION_S` should be used (e.g., when static resources (total resources) are not equal to dynamic resources (available resources)) and `DUMMY_IDLE_DURATION_S` should not trigger scale down? Thanks!

Reply:
Done: https://github.com/ray-project/ray/pull/48519/commits/1e424ce8d6c01093c165d3e3adbed931176b2701